CRAS compile failure

```
name: kali-rolling
encrypted: no
Entering /var/crouton/chroots/kali-rolling...
crouton: version git
release: kali-rolling
architecture: amd64
xmethod: xiwi
targets: xiwi,xfce
host: version 8302.0.0 (Official Build) dev-channel leon 
kernel: Linux localhost 3.8.11 #1 SMP Mon May 9 22:09:11 PDT 2016 x86_64 GNU/Linux
freon: yes
Unmounting /var/crouton/chroots/kali-rolling...
```

#### Please describe your issue:
I just upgraded to Version 52.0.2727.0 dev (64-bit) and I'm receiving the same error as listed in #2572 
```
Compiling CRAS (i386)...
common/cras_config.c: In function 'cras_config_get_system_socket_file_dir':
common/cras_config.c:16:9: error: 'CRAS_SOCKET_FILE_DIR' undeclared (first use in this function)
  return CRAS_SOCKET_FILE_DIR;
         ^
common/cras_config.c:16:9: note: each undeclared identifier is reported only once for each function it appears in
common/cras_config.c:17:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1: all warnings being treated as errors
Failed to complete chroot setup.
```


#### If known, describe the steps to reproduce the issue:
Previous version of ChromeOS did not produce this error.

Seeing this as well when running `crouton -t xfce`.
Using Chrome 52.0.2727.0 (Official Build) dev (64-bit).
```
Compiling CRAS (i386)...
common/cras_config.c: In function 'cras_config_get_system_socket_file_dir':
common/cras_config.c:16:9: error: 'CRAS_SOCKET_FILE_DIR' undeclared (first use in this function)
common/cras_config.c:16:9: note: each undeclared identifier is reported only once for each function it appears in
common/cras_config.c:17:1: error: control reaches end of non-void function [-Werror=return-type]
cc1: all warnings being treated as errors
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/precise...
```

I get this:`Compiling CRAS (i386)...
common/cras_config.c: In function 'cras_config_get_system_socket_file_dir':
common/cras_config.c:16:9: error: 'CRAS_SOCKET_FILE_DIR' undeclared (first use in this function)
  return CRAS_SOCKET_FILE_DIR;
         ^
common/cras_config.c:16:9: note: each undeclared identifier is reported only once for each function it appears in
common/cras_config.c:17:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1: all warnings being treated as errors
Failed to complete chroot setup.`

--- a/targets/audio
+++ b/targets/audio
@@ -151,6 +151,8 @@ END
             convert_automake
 
             echo '
+                CFLAGS="$CFLAGS -DCRAS_SOCKET_FILE_DIR=\"/var/run/cras\""
+
                 buildlib libcras
 
                 # Pass -rpath=$CRASLIBDIR to linker, so we do not need to add

